Units/parser-python.r/python-dot-in-import.d/expected.tags

```diff
@@ -1,12 +1,12 @@
 A.B	input.py	/^from A.B import C as D$/;"	i	roles:namespace
-C	input.py	/^from A.B import C as D$/;"	x	roles:indirectlyImported
-D	input.py	/^from A.B import C as D$/;"	x	roles:def
+C	input.py	/^from A.B import C as D$/;"	x	module:A.B	roles:indirectlyImported
+D	input.py	/^from A.B import C as D$/;"	x	roles:def	nameref:unknown:C
 .	input.py	/^from . import E$/;"	i	roles:namespace
-E	input.py	/^from . import E$/;"	x	roles:imported
+E	input.py	/^from . import E$/;"	x	module:.	roles:imported
 .F	input.py	/^from .F import G$/;"	i	roles:namespace
-G	input.py	/^from .F import G$/;"	x	roles:imported
+G	input.py	/^from .F import G$/;"	x	module:.F	roles:imported
```

Comment:
How about add `scope`field for this test? For `G` it should be `scope:module:.F`.

Reply:
"scope:" field prefix is put only when --fields=+Z is given. Z is not given by default.